db/migrate/20160121192921_create_route_stop_patterns.rb

```diff
@@ -0,0 +1,23 @@
+class CreateRouteStopPatterns < ActiveRecord::Migration
+  def change
+
+    [:current_route_stop_patterns,:old_route_stop_patterns].each do |name|
+      create_table name do |t|
+        t.string :onestop_id
```

Comment:
Probably worth adding a few more indexes, to cover columns that you allow users to query the API by:

- `onestop_id`
- `stop_pattern`

And because it's the side of an association, it's worth adding an index to:

- `created_or_updated_in_changeset_id`